elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/query/DimensionProjection.java

```diff
@@ -11,21 +11,21 @@
 import java.io.Serializable;
 
 /**
- * Represents a selected dimension in a Query.
+ * Represents a selected dimension in a Query.	
```

Comment:
Looks like you added some tabs at the end of the lines?

Reply:
This file is renames to `ColumnProjection`, it would be removed in the following PR. These changes are not important and I don't have this file in my local branch.